protocol/scripts/create_full_node.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
```

Comment:
I would add a comment at the top of this indicating whats going to happen. Just so the user knows what to expect